app/src/main/kotlin/com/mapzen/erasermap/model/LocationClientManager.kt

```diff
@@ -0,0 +1,10 @@
+package com.mapzen.erasermap.model
+
+import com.mapzen.android.lost.api.LostApiClient
+
+interface LocationClientManager {
+
+  abstract fun getClient(): LostApiClient?
```

Comment:
`abstract` qualifier is implied for functions declared in an interface so not required here but no real harm in having it.